_posts/2016/2016-11-21-multitenancy-and-elasticsearch.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: post
+title: "Multitenancy and Elasticsearch"
+date: "2016-11-21 09:00:00"
+image: blog-header/unsplash.jpg
+categories: tech-stories
+authors: ["Ulf S."]
+---
+
+A key requirement for our software stack at ePages is to deal with multitenancy.
+This means that instead of providing an installation for every merchant, we have a single installation serving multiple merchants at the same time.
+
+The first part of this blog post will explain multitenancy with its up- and downsides, the second part will shed some light on solutions when using elasticsearch.
```

Comment:
up- and downsides > better pros and cons?

elasticsearch > in the headline it's written uppercase, on their website I also found it uppercase. Can you unify that within the post?